### DISCUSSION TITLE
Welcome to LaTeX_EnglandWales_Templates Discussions!

### DISCUSSION THREAD

**Discussion by passamo9** [passamo9/LaTeX_EnglandWales_Templates#3] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Giebisch** [comment]
Hey, as I've already stated, great idea !
I've got no knowledge of law in England and Wales, but with proper project planning and some sample documents, like what they look like right now, I can gladly help.

**Reply by passamo9** [reply]
Thanks so much! I'm really happy to work on this together.

I will try to set up the templates in a way that makes it easy for anyone, with any background, able to contribute if they are inclined to do so.

I will try to get more lawyers involved so that we can develop the best format there is to develop!
